clang/lib/DPCT/MigrateScript/MigrateCmakeScript.cpp

```diff
@@ -8,9 +8,9 @@
 #include "MigrateCmakeScript.h"
 #include "Diagnostics/Diagnostics.h"
 #include "ErrorHandle/Error.h"
-#include "UserDefinedRules/PatternRewriter.h"
 #include "FileGenerator/GenFiles.h"
 #include "MigrationReport/Statics.h"
+#include "UserDefinedRules/PatternRewriter.h"
```

Comment:
Can this file be removed? it seems not be used in this file.

Reply:
Clang format automatically applies this. Line 13 is removed from line 10 and added to 13 to maintain alphabetical ordering of includes.